### PR TITLE
Patch anchor links to support CMD+clicks

### LINKED ISSUE #8: Link clicks do not open in new tab
Trying to CMD + click on link within element does not open in new tab, takes over current tab.

I see that you are hijacking links to make the scrolling work.

### PR DESCRIPTION
**Pull request by zerodevx**
Fixes #8.

### CODE REVIEW

File: src/zero-md.js [hunk 1]
```diff
@@ -166,8 +166,12 @@ class ZeroMd extends HTMLElement {
     let path = ev.path || ev.composedPath();
     if (path[0].tagName !== 'A') { return; }
     ev.preventDefault();
-    this._scrollTo(path[0].hash);
-    window.location = path[0].href;
+    if (ev.metaKey) {
+      window.open(window.location.href  + path[0].hash, '_blank');
```
Comment (ernsheong) [inline src/zero-md.js]: Bug here. This is opening the current page in the new tab, not the actual link.

How about let the browser do its thing:

```js
_hijackLinks(ev) {
  let path = ev.path || ev.composedPath();
  if (path[0].tagName !== 'A') { return; }

  const link = path[0];
  if (link.hash && link.href === window.location.href)
    this._scrollTo(path[0].hash);
    ev.preventDefault();
  }
}
```

Comment (ernsheong) [reply src/zero-md.js]: Only `preventDefault` in the special use case of anchor scroll.

Comment (zerodevx) [reply src/zero-md.js]: Good idea, I think that will work. Do you mind opening a PR?

Comment (ernsheong) [reply src/zero-md.js]: Sure, in a bit.

Comment (ernsheong) [reply src/zero-md.js]: Opened https://github.com/zerodevx/zero-md/pull/10

**Pull request merged**